doc/source/raysgd/raysgd_ptl.rst

```diff
@@ -0,0 +1,115 @@
+    train_stats = trainer.train()
+
+And that's it! For a more comprehensive guide, see the MNIST tutorial below.
+
+MNIST Tutorial
+--------------
+In this walkthrough we will go through how to train an MNIST classifier with Pytorch Lightning's ``LightningModule`` and Ray SGD.
+
+We will follow `this tutorial from the PyTorch Lightning documentation
+<https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html>`_ for specifying our MNIST LightningModule.
+
+Setup / Imports
+~~~~~~~~~~~~~~~
+Let's start with some basic imports:
+
+.. literalinclude:: /../../python/ray/util/sgd/torch/examples/pytorch-lightning/mnist-ptl.py
+   :language: python
+   :start-after: __import_begin__
+   :end-before: __import_end__
+
+Most of these imports are needed for building our Pytorch model and training components.
+Only a few additional imports are needed for Ray and Pytorch Lightning.
+
+MNIST LightningModule
+~~~~~~~~~~~~~~~~~~~~~
+We now define our Pytorch Lightning ``LightningModule``:
+
+.. literalinclude:: /../../python/ray/util/sgd/torch/examples/pytorch-lightning/mnist-ptl.py
+   :language: python
+   :start-after: __ptl_begin__
+   :end-before: __ptl_end__
+
+This is the same code that would normally be used in Pytorch Lightning, and is taken directly from `this PTL guide <https://pytorch-lightning.readthedocs.io/en/latest/introduction_guide.html>`_.
+The only difference here is that the ``__init__`` method can optionally take in a ``config`` argument,
+as a way to pass in hyperparameters to your model, optimizer, or schedulers. The ``config`` will be passed in directly from
+the TorchTrainer. Or if using Ray SGD in conjunction with Tune (:ref:`raysgd-tune`), it will come directly from the config in your
+``tune.run`` call.
+
+Training with Ray SGD
+~~~~~~~~~~~~~~~~~~~~~
+We now can define our training function using our LitMNIST module and Ray SGD.
+
+.. literalinclude:: /../../python/ray/util/sgd/torch/examples/pytorch-lightning/mnist-ptl.py
+   :language: python
+   :start-after: __train_begin__
+   :end-before: __train_end__
+
+With just a single ``from_ptl`` call, we can convert our LightningModule to a ``TrainingOperator`` class that's compatible
+with Ray SGD. Now we can take full advantage of all of Ray SGD's distributed trainign features without having to rewrite our existing
+LightningModule.
+
+The last thing to do is initialize Ray, and run our training function!
+
+.. code-block:: python
+
+    # Use ray.init(address="auto") if running on a Ray cluster.
+    ray.init()
+    train_mnist(num_workers=32, use_gpu=True, num_epochs=5)
+
+.. _ptl-unsupported-features:
+
+Unsupported Features
+--------------------
+This integration is currently under active development, so not all Pytorch Lightning features are supported.
+Please post any feature requests on `Github
+<<https://github.com/ray-project/ray/issues>`_ and we will get to it shortly!
+
+A list of unsupported model hooks is as follows:
```

Comment:
you should document when this list was updated.

Reply:
done